app/models/html_publication.rb

```diff
@@ -29,4 +29,8 @@ def copyright_year
   def isbn
     content_store_response.dig("details", "isbn")
   end
+
+  def govspeak_body
```

Comment:
content items already have a `body` attribute this is just this.

Reply:
Thanks, have removed.